README.md

```diff
@@ -138,11 +138,24 @@ All Fairphones after 2 require you to request a code from [this](https://www.fai
 
 Not very fair in my opinion, but whatever fairs your phone.
 
+### Sony
+
+Sony has an open policy for software developers:
+- They are releasing AOSP sources under [Sony Open Devices Program](https://developer.sony.com/open-source/aosp-on-xperia-open-devices).
+- Jolla provides first-class support for installing [Sailfish OS](https://shop.jolla.com/) on select Xperias. That wouldn't be possible without Sony's openness.
+
+But keep in mind it's not perfect:
+- Sony requires you to request an unlock code from [their servers](https://developer.sony.com/open-source/aosp-on-xperia-open-devices/get-started/unlock-bootloader). They can pull the plug anytime. You have to send your IMEI and agree to warranty voiding.
```

Comment:
The openness is awesome, but I think they still deserve some more slack for the server requirement. I'll have to think about that.